### DISCUSSION TITLE
Nightly with AR encryption

### DISCUSSION THREAD

**Discussion by Umiamz** [mastodon/mastodon#29995] [Q&A] [answered]
I updated to the latest nightly (2024-04-18) and had to add the AR environment variables for the precompile to run. However, it was late, I was tired, etc. etc. and I didn't read the warning properly so didn't generate my own keys and used the ones from .env.test instead. 

Is it too late to generate my own keys with db:encryption:init? Will anything in the database have been encrypted yet? I didn't see any output when I ran the pre- and post-deployment db:migrate.

This is a single-user instance, by the way.

Thanks in advance!

**Comment by renchap** [comment, marked as answer]
We added support for AR Encryption but nothing is using it yet. Feel free to change the keys for now!

**Reply by Umiamz** [reply]
Thanks for the reply - much appreciated!
